tuxemon/core/monster.py

```diff
@@ -50,7 +50,7 @@
     'slug',
     'status',
     'total_experience',
-    'flairs'
+    'flairs',
```

Comment:
why the trailing comma? is this not incorrect?

Reply:
im going to remove this.  it was just a knee-jerk reaction that i made.  i prefer trailing commas on all items because it makes it really easy to add items or rearrange without maintaining the last comma.  eventually, i will run black on the entire code base, which will add trailing commas as part of its style guidelines.

Reply:
to clarify, i will remove it because its not related to the change of the pr